docs/zh-CN/developer-guide/fe-idea-dev.md

```diff
@@ -37,9 +37,22 @@ JDK1.8+, IntelliJ IDEA
 3. 如果仅进行fe开发而没有编译过thirdparty，则需要安装thrift，并将thrift 复制或者连接到 `thirdparty/installed/bin` 目录下
 
         安装 `thrift` 0.13.0 版本(注意：`Doris` 0.15 和最新的版本基于 `thrift` 0.13.0 构建, 之前的版本依然使用`thrift` 0.9.3 构建)
+
+        Windows: 
+           ① 下载：`http://archive.apache.org/dist/thrift/0.13.0/thrift-0.13.0.exe`
```

Comment:
You may be not familiar with markdown.
It's better use 1. instead of ①.
```suggestion
           1. 下载：`http://archive.apache.org/dist/thrift/0.13.0/thrift-0.13.0.exe`
```

Reply:
The following also